roles/network/tasks/install.yml

```diff
@@ -7,7 +7,7 @@
   package:
     name: networkd-dispatcher    # 15kB download: Dispatcher service for systemd-networkd connection status changes
     state: present
-  when: not is_raspbian or not is_linuxmint
+  when: not is_linuxmint
```

Comment:
Can this line (`when: not is_linuxmint`) now be removed, as discussed during the call (http://minutes.iiab.io) earlier today?